src/nvidiautil@ethanwharris/processorHandler.js

```diff
@@ -21,35 +22,37 @@ const Me = ExtensionUtils.getCurrentExtension();
 const Processor = Me.imports.processor;
 
 var ProcessorHandler = class {
-  constructor() {
-    this._processors = [false, false, false];
-  }
-  process() {
-    for (let i = 0; i < this._processors.length; i++) {
-      if (this._processors[i]) {
-        try {
-          this._processors[i].process();
-        } catch (err) {
-          Main.notifyError("Error parsing " + this._processors[i].getName(), err.message);
-          this._processors[i] = false;
+    constructor() {
+        this._processors = [false, false, false];
+    }
+
+    process() {
+        for (let i = 0; i < this._processors.length; i++) {
+            if (this._processors[i]) {
+                try {
+                    this._processors[i].process();
+                } catch (err) {
+                    Main.notifyError(`Error parsing ${this._processors[i].getName()}`, err.message);
+                    this._processors[i] = false;
+                }
+            }
         }
-      }
     }
-  }
-  addProperty(property, listeners) {
-    let processor = property.declare();
-    if (!this._processors[processor]) {
-      this._processors[processor] = new Processor.LIST[processor]();
+
+    addProperty(property, listeners) {
+        let processor = property.declare();
+        if (!this._processors[processor])
+            this._processors[processor] = new Processor.LIST[processor]();
```

Comment:
it makes this block particularly hard to read

Reply:
I agree! The name is useless, the square brackets already tell it's a list :man_shrugging: